pkg/brownfield/http_settings_test.go

```diff
@@ -6,13 +6,28 @@
 package brownfield
 
 import (
+	"strings"
+
+	n "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-12-01/network"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
 	ptv1 "github.com/Azure/application-gateway-kubernetes-ingress/pkg/apis/azureingressprohibitedtarget/v1"
 	"github.com/Azure/application-gateway-kubernetes-ingress/pkg/tests/fixtures"
 )
 
+type mockLogger struct {
```

Comment:
is this supposed to be in a different file ?

Reply:
It would actually make sense to move it somewhere in the pkg/tests namespace

Reply:
Moved to `pkg/tests/mocks`